shared/stories/index.desktop.js

```diff
@@ -12,6 +12,17 @@ import {initDesktopStyles} from '../styles/index.desktop'
 
 const stories = {...sharedStories, ...desktopStories}
 
+const filter = null
```

Comment:
Useful for when developing to only show the story you're working on. Also makes reloads faster.